### PR TITLE
feat: add vergrendeld policy

### PR DESCRIPTION
**Pull request by RickWoltheus**
Add vergreneld policy

Solves PZ-1288

### CODE REVIEW

File: docs/solution-architecture/accessControlPolicies.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ various resources on which a user can perform actions:
...
 | lezen                                         |                                          X                                           |      X      |       X       |               | OPA policy `document lezen`                                      |
 | wijzigen                                      | X (zaak open, document onvergrendeld, niet definitief of vergrendeld door gebruiker) |             |       X       |               | OPA policy `document wijzigen`                                   |
-| verwijderen                                   |                                          X                                           |             |       X       |               | OPA policy `document verwijderen`                                |
+| verwijderen                                   |                      X (Document onvergrendeld door gebruiker)                       |             |       X       |               | OPA policy `document verwijderen`                                |
```
Comment (edgarvonk) [inline docs/solution-architecture/accessControlPolicies.md:89]: What does `onvergrendeld door gebruiker` mean? I would think `onvergrendeld` is enough or not? Does it really matter that it was the current user who did the unlocking?

Comment (edgarvonk) [reply docs/solution-architecture/accessControlPolicies.md:89]: This is also what `document.vergrendeld == false` means: simply: `document onvergrendeld`

**Review by edgarvonk — CHANGES_REQUESTED** [review]

File: docs/solution-architecture/accessControlPolicies.md (hunk 1 above)
Comment (RickWoltheus) [reply docs/solution-architecture/accessControlPolicies.md:89]: I followed the convention of the rest of the document, but I agree we could make it onvergrendeld

Comment (edgarvonk) [reply docs/solution-architecture/accessControlPolicies.md]: Well, e.g. `vergrendeld door gebruiker` really means: locked _by the current user trying to change this document_`. Whereas `onvergrendeld` simply means: unlocked, no matter by whom. So it is really different.

**Review by edgarvonk — APPROVED** [review]
nice one!

**Pull request merged**